### ISSUE TITLE
Outputting the value of a select box ACF

### ISSUE THREAD

**Issue by frazerf** [timber/timber#1524]
I'm wanting to output the value of a select box but for some reason it's not playing ball. 

I've called the field describe_yourself and there are 7 options they can select from. 

I've tried - {{author.describe_yourself}} and {{author.describe_yourself.value}} but both of them don't seem to work. Has anyone had any experience with pulling through the value of select box?

**Comment by gchtr** [comment]
Do you use Advanced Custom Fields for that select box? If yes, can you try and see if it works with 
`{{ author.get_field('describe_yourself') }}`

When you say that they *don’t seem to work*, what exactly doesn’t? Do you get an unexpected output, or no output at all?

**Comment by frazerf** [comment]
Hi @gchtr ,

Yes I'm using the select box from ACF. And I've set the return format to - value.

I've tried your recommendation above, but again it's showing no output whatsoever. You don't have any other suggestions do you? I've been scouring stackoverflow for any sort of documentation of timber and acf selectbox's but nothing is showing. 

Thanks

**Comment by gchtr** [comment]
Alright. I just tried it. You should be able to use `{{ author.describe_yourself }}` **if `author` is an instance of `Timber\Post` or `Timber\User`**. If `author` is a `WP_Post` or `WP_User` object, it won’t work.

Can you show the PHP code where you add `author` to `$context`, before it is passed to the Twig file?

**Comment by frazerf** [comment]
Sorry this isn't formatted. I thought you might have been on to something with the WP_User object but it is using Timber\User. 

`<?php
/**
 * The template for displaying Author Archive pages
 *
 * Methods for TimberHelper can be found in the /lib sub-directory
 *
 * @package  WordPress
 * @subpackage  Timber
 * @since    Timber 0.1
 */
global $params;
global $wp_query;

$context = Timber::get_context();

$auth = $params['author'];

$context['moments'] = Timber::get_posts(array(
	'post_type'		=> 'moment',
	'posts_per_page'   => -1,
	'author_name'	   =>  $auth,
));

$context['rotations'] = Timber::get_posts(array(
	'post_type'      => 'rotation',
	'posts_per_page' => -1,
	'author_name'    => $auth,
	'order'          => 'ASC',
	'orderby'				 => 'date'
), 'RotationMoments');

$getAuthor = get_user_by('slug', $auth);

$author = new TimberUser( $getAuthor->ID );
$context['author'] = $author;
$context['title'] = $author->name();

Timber::render( array( 'author-profile.twig', 'archive.twig' ), $context );

`

**Comment by gchtr** [comment]
And I assume you’re using the latest version of Timber and WordPress?

I checked again, when I use `$author = new TimberUser( 1 );`, it correctly sets the selected value as an object property. (Btw, `TimberUser()` is deprecated, the new classes are namespaced, so it would be `Timber\User()`).

Can you do a `var_dump( $author );` right after `$author = new TimberUser( $getAuthor->ID );` and see if at least the `describe_yourself` property is set on `$author`? It might be `false`, so you wouldn’t get any output with Twig.

Another reason that might cause this to fail are other plugins that filter ACF output. You could deactivate all other plugins except ACF and Timber and see if that has any effect.

Filters might also be defined somewhere in the theme.

**Comment by frazerf** [comment]
Yeah you're right there. I've just done a `var_dump( $author );` and describe_yourself isn't even showing on the $author. Is there some way I can set this to true? Thanks again for your help. Really appreciate it!

**Comment by gchtr** [comment]
Hey @frazerf, sorry for the late reply here, I’ve been rather busy. Were you able to solve this issue yet?
